packages/core/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -237,10 +234,7 @@ ChoiceList.propTypes = {
    */
   onComponentBlur: PropTypes.func,
   onChange: PropTypes.func,
-  /**
-   * If the component renders a select, set the max-width of the input either to `'small'` or `'medium'`.
-   */
-  size: PropTypes.oneOf(['small', 'medium']),
```

Comment:
I would maybe add a description like `Sets the size of the checkbox or radio button to be small`

Reply:
I added the comment!